motioneye/controls/smbctl.py

```diff
@@ -305,11 +305,11 @@ def _check_mounts():
 
     logging.debug('checking SMB mounts...')
 
-    stop, start = update_mounts()
-    if stop:
+    stop_mounts, new_start = update_mounts()
+    if stop_mounts:
```

Comment:
Better use the same variable names used `update_mounts`, which also better indicate their meaning: whether `motion` needs to be stopped before SMB shares are unmounted, and whether it needs to be started after SMB shares have been (re)mounted:
```suggestion
    should_stop, should_start = update_mounts()
    if should_stop:
```